### PR TITLE
Only show "delay" type alerts on bus page

### PR DESCRIPTION
**Pull request by ErinLMoore**
We were displaying all kinds of alerts, this restricts to just alerts with an effect of "delay".

Asana task: https://app.asana.com/0/1167196461144361/1204992413147837/f

### CODE REVIEW

File: lib/alerts_viewer_web/live/bus_live.ex [hunk 1]
```diff
@@ -18,7 +18,13 @@ defmodule AlertsViewerWeb.BusLive do
     current_algorithm = hd(delay_alert_algorithm_components)
 
     bus_routes = Routes.all_bus_routes()
-    bus_alerts = if(connected?(socket), do: Alerts.subscribe() |> filtered_by_bus(), else: [])
+
+    bus_alerts =
+      if(connected?(socket),
+        do: Alerts.subscribe() |> filtered_by_bus() |> filtered_by_delay_type(),
```
Comment (arkadyan) [inline lib/alerts_viewer_web/live/bus_live.ex:24]: suggestion: I believe that there's one additional place where we are filtering for bus alerts and that we also need to add `filtered_by_delay_type` to. That's when we get new alerts coming in down in `handle_info({:alerts, alerts}`.

Comment (ErinLMoore) [reply lib/alerts_viewer_web/live/bus_live.ex:24]: good catch!

File: lib/alerts_viewer_web/controllers/bus_controller.ex [hunk 2]
```diff
@@ -9,7 +9,7 @@ defmodule AlertsViewerWeb.BusController do
     mod = String.to_existing_atom(algorithm)
     bus_routes = Routes.all_bus_routes()
     stats_by_route = RouteStatsPubSub.all()
-    bus_alerts = Alerts.all()
+    bus_alerts = Alerts.all() |> Alerts.by_service("3") |> Alerts.by_effect("delay")
```
Comment (arkadyan) [inline lib/alerts_viewer_web/controllers/bus_controller.ex:12]: 💡 I didn't think of this one. But is it already covered by the `delay_alert?` check on the following line?

**Review by arkadyan — APPROVED** [review]
One small question but otherwise looks good.

File: lib/alerts_viewer_web/controllers/bus_controller.ex (hunk 2 above)
Comment (ErinLMoore) [reply lib/alerts_viewer_web/controllers/bus_controller.ex:12]: Oh, yeah, unlike the other file that is the only place that it's used

**Pull request merged**